python_labs/exam_test.py

```diff
@@ -0,0 +1,14 @@
+# тут я попытался написать хоть какие-то тесты, но почему-то получаю ошибку 'str' object is not callable
+
+import unittest
+import exam
+
+class TestExam(unittest.TestCase):
+
+    def test_a_is_palindrome(self):
+        assert exam.a("A") is True
```

Comment:
```suggestion
        self.assertEqual(True, exam.checkPalindrome("A"))
```
ну и лучше 1 тест - 1 assert